mzLib/Readers/Readers.csproj

```diff
@@ -1,17 +1,19 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
 		<TargetFramework>net8.0</TargetFramework>
 		<Platforms>x64</Platforms>
 		<ImplicitUsings>enable</ImplicitUsings>
 		<Nullable>enable</Nullable>
+		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
"In a C# project file (.csproj), the "AllowUnsafeBlocks" property within a "PropertyGroup" is used to enable the compilation of code containing "unsafe" blocks, which allows the use of pointers and direct memory manipulation, but should be used with caution due to the potential for memory management issues and security risks."